src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -582,6 +582,10 @@ public String getMetricsCollectorClass() {
         return getString("secor.monitoring.metrics.collector.class");
     }
     
+    public Map<String, String> getCustomTopicsNames() {
+    	return getPropertyMapForPrefix("secor.kafka.customtopicname");
```

Comment:
What's the behavior if people didn't specify that parmater in the property file?